plugins/rts54hub/rts54hub.quirk

```diff
@@ -38,3 +38,24 @@ FirmwareSizeMax = 0x90000
 Rts54TargetAddr = 0x20
 Rts54I2cSpeed = 0x2
 Rts54RegisterAddrLen = 0x04
+
+# Acer D501 Dock
+[USB\VID_2BEF&PID_1009]
+Plugin = rts54hub
+GType = FuRts54HubDevice
+Vendor = Realtek
```

Comment:
Not Acer?

Reply:
This VID is QSI( Quanta Storage Inc) which is ODM of Acer Dock
Acer do not customized ID on this project

Reply:
Okay, so why do we need the Realtek override? Usually we just set the vendor automatically based on the VID.

Reply:
okay, so just don't set the Vendor default is automatic?

Reply:
Yes, I think you can just remove the `Vendor=` line completely.